x/tokenomics/keeper/keeper_test.go

```diff
@@ -209,6 +209,53 @@ func (s *TestSuite) TestClaimSettlement_ClaimSettled_ProofRequiredAndProvided_Vi
 	require.Equal(t, s.numComputeUnits, expectedEvent.ComputeUnits)
 }
 
+func (s *TestSuite) TestClaimSettlement_ClaimSettled_ProofRequiredAndProvided_ViaProbability() {
+	// Retrieve default values
+	t := s.T()
+	ctx := s.ctx
+	sdkCtx := sdk.UnwrapSDKContext(ctx)
+	sharedParams := s.keepers.SharedKeeper.GetParams(ctx)
+
+	// Set the proof parameters to require a proof via probability AND NOT via threshold.
```

Comment:
I rewrote this test to hold the claim's compute units constant, at a value that is equal to the threshold, therefore requiring a proof by default. In test cases where we don't want to require a proof, we modify the proof module params instead. I felt like this would be easier to follow and maintain.

Reply:
I've updated the comments. PTAL and LMK WDYT.